core/src/main/kotlin/no/strooped/TextureSizes.kt

```diff
@@ -22,4 +22,16 @@ object TextureSizes {
         // pinLabel on JoinGameScreen
         fun pinLabelWidth(): Float = Gdx.graphics.width * (115f / 1080f)
         fun pinLabelHeight(): Float = Gdx.graphics.height * (64f / 2040f)
+
+        // loadCircle on LobbyScreen
+        fun loadWidth(): Float = Gdx.graphics.width * (130f / 1080f) // 105
```

Comment:
Do you «load» width or «get» width? Also instead of using comments to tell what the code belongs to, do I suggest coupling properties related to circle togetherm, properties related to pinLabel together, etc. Either through custom components, nested classes or something else. But it will then make it much more easier to understand the context when using these values

Reply:
loadWidth refers to loading circle width. I can expand the naming. We can talk about suggestions for the second part in the call

Reply:
Done